spec/models/initiative_spec.rb

```diff
@@ -17,13 +17,20 @@
       end
     end
   end
-end
 
-describe InitiativesController, type: :controller do
-  let!(:initiative) { create(:initiative) }
+  describe InitiativesController, type: :controller do
```

Comment:
Check that you're describing the right thing, and the right type. Compare with the spec above.